package.json

```diff
@@ -75,9 +75,10 @@
     "postbuild": "npm-run-all --parallel copy:*",
     "postinstall": "node postinstall.js",
     "start": "node build/server.js",
-    "test": "npm run test:server",
+    "test": "npm run test:server; npm run test:client",
```

Comment:
Details, but you can use `npm-run-all 'test:*'" to serialize the run of all `test:`scripts #protip